### PR TITLE
Update index.d.ts

### PR DESCRIPTION
**Pull request by DenisCarriere**
Updates from new method offset

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -19,6 +19,7 @@ declare namespace cheapRuler {
         bufferPoint(p: Point, buffer: number): BBox;
         bufferBBox(bbox: BBox, buffer: number): BBox;
         insideBBox(p: Point, bbox: BBox): boolean;
+        offset(p: Point, dx: number, dy: number): Point:
```
Comment (mourner) [inline index.d.ts]: typo here — semicolon

Comment (DenisCarriere) [inline index.d.ts]: Ops :) I did the PR using the browser

**Comment by DenisCarriere** [comment]
Fixed

**Comment by mourner** [comment]
Bummer, forgot to update this file before releasing 2.5.0. Does this fix warrant a 2.5.1 or is it fine for now?

**Comment by DenisCarriere** [comment]
No rush, next patch/minor release is fine.

**Comment by DenisCarriere** [comment]
No rush, next patch/minor release is fine.

**Pull request merged**